api/models/model.py

```diff
@@ -151,6 +156,19 @@ def getByDeploymentStatus(self, deployment_status):
             .all()
         )
 
+    def getByPublishStatus(self, publish_status):
+        return self.dbs.query(Model).filter(Model.is_published == publish_status).all()
+
+    def getByNotLastUsedWithinWeek(self):
+        one_week_ago = datetime.utcnow() - timedelta(weeks=1)
+        return (
+            self.dbs.query(Model)
+            .filter(
+                db.or_(Model.last_used < one_week_ago, Model.last_used == None)
```

Comment:
Just to verify the conditionals:
(1) `Model.last_used < one_week_ago` means the model was NOT last used between [now - week, now]

(2) `Model.last_used == None` is because the `last_used` column starts off as `NULL`

I have to change (2), because when we run this migration, everything row in `models` will set `last_used=NULL`, so everything will be grounds for removal. 

I added (2) in the first place because if the model is never used from the time this PR is merged and deployed onto prod, the column will always remain `NULL` and will never be removed (even if no one is interacting with it).

I'm not sure what the best way to solve this is, maybe setting the `last_used` column to `now()` in the migration script, so that when this gets deployed, nothing will be removed (at least for the first week), and then gradually models that have not been used after a week we deploy this change will be removed.